en/scripting/modules/index.md

```diff
@@ -1,88 +1,86 @@
-# Module
+# Modules
 
-Engine and editor expose their functionalities API through modules. Modules are in form of ECMAScript module format.
+Engines and editors expose their functionality through API interfaces and modules that exist in the form of **ECMAScript** modules.
 
-⚠️ Note, starting from 3.0, you can not access API through global variable `cc`!
+> **Note**: starting from 3.0, engine functions cannot be accessed through the global variable, prefixed with `cc`!
 
-## Engine modules
+## Engine Module
 
-At present, engine only offers one public module `'cc'`.
+Currently, the engine only provides a public module prefixed with `'cc'`.
 
-Contents of module `'cc'` are dynamically decided, which is relevant with **Engine Module** setting in **Project Setting** panel.
+The content of the `'cc'` module is dynamic, and its content is related to the setting of **engine module** in **Project Settings**.
 
-### Engine logging
+### Engine log output
 
-Example:
+Examples are as follows:
 
 ```ts
-import { log } from 'cc';
+import {log} from'cc';
 log(`Hello world!`);
 ```
 
-## Editor modules
+## Editor Module
 
-Editor modules are under protocol `'cce:'` ("cce" is abbreviation of "**C**ocos**C**reator**E**ditor").
+The editor modules are all under the `'cce:'` protocol (**cce** stands for "**C**ocos**C**reator**E**ditor").
 
-Except for module `cce.env`, all modules are visible only under editor environments. For example, they are not visible from previewing and after building. Instead, they are visible from scene editor.
+With the only exception being `'cce.env'`, all editor modules are only available in the editor environment. For example, the editor module cannot be accessed in the preview and build environment, on the contrary, it can be accessed in the **scene editor**.
```

Comment:
1. being \`\'cce.env'\` -> being module \`cce.env\`
2. in the preview and build environment -> in the environment after preview and build
3. \*\*scene editor\*\* -> \*\*Scene\*\* panel

Reply:
1. With `'cce.env'` being the only exception,
2. in the environment after previewing and building
3. ok